sync/clients/sync.py

```diff
@@ -122,6 +156,13 @@ def get_project_recommendation(self, project_id: str, recommendation_id: str) ->
             )
         )
 
+    def get_latest_project_recommendation(self, project_id: str) -> dict:
```

Comment:
we should know what this request response object looks like right? May be worth updating this `dict` to a project recommendation schema

Reply:
For the API calls, I've been following the pattern of the other calls, and that's why it's a `dict`. I agree we should have a schema for this, but it might make more sense to have a large overhaul of the API response types